test/apps/iodemo/io_demo.cc

```diff
@@ -100,6 +100,9 @@ typedef struct {
         UcxLog(LOG_PREFIX, !(_expression), &std::cerr, do_assert) \
                 << ASSERTV_STR(#_expression)
 
+int set_sockaddr(const char *ip_str,
+                 uint16_t port,
+                 struct sockaddr *addr);
```

Comment:
here and at definition - pls wrap up arg list by line length (80 symbols)
(but I'm not sure if this function is actually needed)

Reply:
fixed